### PR TITLE
Update README.markdown

### PR DESCRIPTION
**Pull request by yulonghu**
Update README.markdown

Redis::close()

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
 
 ### close
 -----
-_**Description**_: Disconnects from the Redis instance, except when `pconnect` is used.
+_**Description**_: Disconnects from the Redis instance, include when `pconnect` is used.
+
+*Note*: Requires Redis >= 4.2.0, phpredis can closed the persistent connection.
```
Comment (yatsukhnenko) [inline README.markdown]: I'm not an expert bu may be `can close` instead `can closed`?

Comment (yulonghu) [reply README.markdown]: updated

File: README.markdown [hunk 2]
```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
 
 ### close
 -----
-_**Description**_: Disconnects from the Redis instance, except when `pconnect` is used.
+_**Description**_: Disconnects from the Redis instance, include when `pconnect` is used.
```
Comment (jrchamp) [inline README.markdown]: > include when

including when

File: README.markdown [hunk 3]
```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
...
+
+*Note*: Requires Redis >= 4.2.0, phpredis can close the persistent connection.
```
Comment (jrchamp) [inline README.markdown]: > Requires Redis >= 4.2.0, phpredis can close the persistent connection.

Redis >= 4.2.0 is required to close a persistent connection.

**However, I prefer:**

Closing a persistent connection requires Redis >= 4.2.0.

File: README.markdown (hunk 2 above)
Comment (jrchamp) [reply README.markdown]: Alternatively, now that pconnect works also, you can remove the clause entirely. End result:

> Disconnects from the Redis instance.

**Comment by michael-grunder** [comment]
Merged in initial changes and incorporated verbiage update from @jrchamp 

Thanks everyone!
Cheers,

**Pull request merged**